demisto_sdk/tests/validate_content_path_test.py

```diff
@@ -31,6 +33,31 @@
 )
 
 
+@pytest.mark.parametrize(
+    "templates_folder, template_name",
+    (
+        ("template_foo", "template_foo.json"),
+        ("template_foo", "template_foo.yml"),
+    ),
+)
+def test_xdrc_template_file_valid(templates_folder: str, template_name: str):
+    _validate(DUMMY_PACK_PATH / XDRC_TEMPLATE_DIR / templates_folder / template_name)
+
+
+@pytest.mark.parametrize(
+    "templates_folder, template_name",
+    (
+        ("template_foo", "template_foo_test.json"),
+        ("template_foo", "template_foo_test.yml"),
+    ),
+)
+def test_xdrc_template_file_invalid(templates_folder: str, template_name: str):
```

Comment:
Let's test a capital letter, trailing underscore, just to make it a tiny bit clearer what this test is for (took me a minute to figure out the `_test` was the punch here)